cli-plugins/apps/getting-started/installation.hbs.md

```diff
@@ -0,0 +1,50 @@
+# Installation
+
+## Prerequisites
+
+Ensure that you installed or updated the core Tanzu CLI. For more information, see
+[Install Tanzu CLI](https://github.com/vmware-tanzu/tanzu-cli/blob/main/docs/quickstart/install.md).
+
+## Install Apps CLI Plug-in
+
+From VMware Tanzu CLI
+: Install Apps CLI plug-in from TAP group:
+
+```bash
+tanzu plugin install apps --group vmware-tap/tap:<version>
```

Comment:
typo in the group name.
it should be
`tanzu plugin install apps --group vmware-tap/default:<version>`

Reply:
fixed